### PR TITLE
Create Plugin_SimpleMapDetails.as

### PR DESCRIPTION
**Pull request by ghost**
Displays Map-Details like Author, Times etc...

### CODE REVIEW

File: Plugin_SimpleMapDetails.as [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#name "SimpleMapDetails"
+#author "derfuhr"
+
+bool windowIsVisible = false;
+
+void Main(){
```
Comment (codecat) [inline Plugin_SimpleMapDetails.as]: You don't need a `Main()` if you have the other functions already doing stuff. :+1:

**Review by codecat — CHANGES_REQUESTED** [review]

File: Plugin_SimpleMapDetails.as (hunk 1 above)
Comment (ghost) [reply Plugin_SimpleMapDetails.as]: Ok, good to know.

**Review by codecat — APPROVED** [review]

**Comment by codecat** [comment]
It would be cool to also add comments to this to explain what exactly is happening in the code, but this looks fine for now. :+1:

**Comment by ghost** [comment]
Yes ok, i will add some comments next time. ;)

**Pull request merged**